plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/AbstractXmlSourceValidatorTest.java

```diff
@@ -132,11 +143,12 @@ public void testGetFile() throws CoreException {
 
     assertNotNull(testFile);
     assertEquals(file, testFile);
+    file.delete(true, null);
```

Comment:
May want to put this in a finally block so it happens even if an exception is thrown.